weaviate/connect/base.py

```diff
@@ -132,21 +132,24 @@ def _grpc_channel(self, async_channel: bool) -> Union[Channel, AsyncChannel, Non
             return None
 
         if async_channel:
-            import_path = grpc.aio
-        else:
-            import_path = grpc
-
-        if self.grpc.secure:
-            return import_path.secure_channel(
-                target=self._grpc_target,
-                credentials=ssl_channel_credentials(),
-                options=GRPC_OPTIONS,
+            return AsyncChannel(
+                host=self.grpc.host,
+                port=self.grpc.port,
+                ssl=self.grpc.secure,
+                # config=GRPC_OPTIONS,
```

Comment:
commented out?

Reply:
Oh yes, that option doesn't exist in the `grpclib` package. I'll remove the comment but hopefully there is a way to customise message sizes, I'll check